shenyu-spring-boot-starter/shenyu-spring-boot-starter-instance/src/main/java/org/apache/shenyu/springboot/starter/instance/ShenyuInstanceConfiguration.java

```diff
@@ -39,4 +41,19 @@ public class ShenyuInstanceConfiguration {
     public InstanceRegisterListener instanceRegisterListener(final ShenyuConfig config) {
         return new InstanceRegisterListener(config.getInstance());
     }
+
+    /**
+     * ShenYu Instance Register Repository.
+     * expose to shenYu-sdk.
+     *
+     * @param config the config
+     * @return ShenYu Instance Register Repository
+     */
+    @Bean
+    @ConditionalOnProperty(name = "shenyu.instance.clientEnable", havingValue = "true")
```

Comment:
whats mean？ 

ShenyuInstanceRegisterRepository already init in InstanceRegisterListener

Reply:
InstanceRegisterListener will register the service to the registry, shenyu-sdk does not need this operation.

Reply:
Yes， so this code maybe in spring-boot-starter-sdk

Reply:
I get it, I'll revise it